src/Service/Modules/BomVerificationHistoryModule.cs

```diff
@@ -19,14 +19,9 @@ public class BomVerificationHistoryModule : ICarterModule
     {
         public void AddRoutes(IEndpointRouteBuilder app)
         {
-            app.MapGet("/purchasing/bom-verification/", this.GetApp);
             app.MapGet("/purchasing/bom-verification/{id:int}", this.GetBomVerificationHistoryEntry);
-            app.MapPost("/purchasing/bom-verification/", this.CreateBomVerificationHistoryEntry);
-        }
-
-        private async Task GetApp(HttpRequest req, HttpResponse res)
-        {
-            await res.Negotiate(new ViewResponse { ViewName = "Index.html" });
+            app.MapPost("/purchasing/bom-verification/create", this.CreateBomVerificationHistoryEntry);
```

Comment:
Why add the /create here?

Reply:
Was looking at similar parts of App and that seemed to be a kind of running theme. Happy to change it back though!